[ja_JP] Translate "peerchecklist.md" into Japanese (#755)
hyperledger/fabric-docs-i18n: docs/locale/ja_JP/source/deploypeer/peerchecklist.md

chaincodeListenAddress: 0.0.0.0:7052
```

> - **`chaincodeListenAddress`**: (デフォルト値を上書きする必要があります。) このパラメータのコメントアウトを外し、このピアがチェーンコード要求をリッスンするアドレスを指定します。`peer.listenAddress` とは異なる必要があります。例えば、 `0.0.0.0:7052` です。

Reviewer: ```suggestion
- **`chaincodeListenAddress`**: (デフォルト値を上書きする必要があります。) このパラメータのコメントアウトを外し、このピアがチェーンコード要求をlistenするアドレスを指定します。`peer.listenAddress` とは異なる必要があります。例えば、 `0.0.0.0:7052` です。
```
listenについて前と表記ゆれがあるので、どちらかに統一したほうがよいでしょう。

# When used as CLI config, this means the peer's endpoint to interact with
address: 0.0.0.0:7051
```
- **`address`**: (デフォルト値を上書きする必要があります。) 組織内の他のピアがこのピアに接続するために使用するアドレスを指定します。例えば、 `peer0.org1.example.com:7051` です。

## peer.mspConfigPath
```
mspConfigPath: msp
```

> - **`mspConfigPath`**: (デフォルト値を上書きする必要があります。) これはピアのローカルMSPへのパスであり、ピアをデプロイする前に作成する必要があります。パスは、 `FABRIC_CFG_PATH` に対する絶対パスでも相対パスでもかまいません(デフォルトでは、ピアイメージにある `/etc/hyperledger/fabric` です)。「msp」以外の名前が付けられたフォルダへの絶対パスが指定されていない限り、ピアはデフォルトでそのパス(つまり `FABRIC_CFG_PATH/msp` )で「msp」という名前のフォルダを探し、ピアイメージを使用する場合は `/etc/hyperledger/fabric/msp` になります。[Registering and enrolling identities with a CA](https://hyperledger-fabric-ca.readthedocs.io/en/release-1.4/deployguide/use_CA.html) トピックで説明されている推奨フォルダ構造を使用している場合、FABRIC_CFG_PATHに対する相対パスは次のようになります。

Reviewer: ```suggestion
- **`mspConfigPath`**: (デフォルト値を上書きする必要があります。) これはピアのローカルMSPへのパスであり、ピアをデプロイする前に作成する必要があります。パスは、絶対パスでも `FABRIC_CFG_PATH` に対する相対パスでもかまいません(デフォルトでは、ピアイメージにある `/etc/hyperledger/fabric` です)。「msp」以外の名前が付けられたフォルダへの絶対パスが指定されていない限り、ピアはデフォルトでそのパス(つまり `FABRIC_CFG_PATH/msp` )で「msp」という名前のフォルダを探し、ピアイメージを使用する場合は `/etc/hyperledger/fabric/msp` になります。[Registering and enrolling identities with a CA](https://hyperledger-fabric-ca.readthedocs.io/en/release-1.4/deployguide/use_CA.html) トピックで説明されている推奨フォルダ構造を使用している場合、FABRIC_CFG_PATHに対する相対パスは次のようになります。
```

  - **`useLeaderElection:`** (v2.2のデフォルトは `false` です。これはピアがオーダリングサービスからブロックを取得するために推奨されます。) `useLeaderElection` がfalseに設定されている場合、 `peer.gossip.orgLeader` をtrueに設定して、少なくとも1つのピアを組織のリーダーにする必要があります。組織内のピア間でブロック配布にゴシップを使用するようにピアを設定する場合は、 `useLeaderElection` をtrueにします。

  - **`orgLeader:`** (v2.2のデフォルトは `true` です。これはピアがオーダリングサービスからブロックを取得するために推奨されます。) 組織内のピア間でブロック配布にゴシップを使用する場合は、この値を `false` に設定します。

  - **`state.enabled:`** (v2.2のデフォルトは `false` です。これはピアがオーダリングサービスからブロックを取得するために推奨されます。) ゴシップを使用して欠落したブロックを同期する場合は、この値を `true` に設定します。これにより、遅れているピアがネットワーク上の他のピアに追いつくことができます。

* **Implicit data**  Fabric v2.0では、ピア上のプライベートデータ暗黙的なコレクションの概念が導入されました。組織ごとのプライベートデータパターンを利用したい場合は、チェーンコードをFabric v2.*でデプロイするときはコレクションを定義する必要はありません。暗黙的な組織固有のコレクションは、事前の定義なしで使用できます。この新しい機能を利用する場合は、 `pvtData.implicitCollectionDisseminationPolicy.requiredPeerCount` と `pvtData.implicitCollectionDisseminationPolicy.maxPeerCount` の値を設定する必要があります。詳細は、 [Private data tutorial](../private_data_tutorial.html) をご覧ください。

>   - **`pvtData.implicitCollectionDisseminationPolicy.requiredPeerCount`:** (プライベートデータの暗黙的コレクションを使用する場合は、この値を上書きすることをお勧めします。) **New in Fabric 2.0.** デフォルトは0ですが、組織に属するピアの数に基づいて増やす必要があります。この値は、トランザクションをエンドースした後にピアがダウンした場合にデータの冗長性を確保するために、データを配布する必要がある自分の組織内のピアのトランザクション数を表します。

Reviewer: ```suggestion
  - **`pvtData.implicitCollectionDisseminationPolicy.requiredPeerCount`:** (プライベートデータの暗黙的コレクションを使用する場合は、この値を上書きすることをお勧めします。) **New in Fabric 2.0.** デフォルトは0ですが、組織に属するピアの数に基づいて増やす必要があります。この値は、トランザクションをエンドースした後にピアがダウンした場合にデータの冗長性を確保するために、データを配布する必要がある自分の組織内のピアの数を表します。
```

      - tls/ca.crt
```

> ピアに対してTLSコミュニケーションを有効にするために、このセクションを設定します。TLSが有効になった後は、ピアと取り引きをする全てのノードも、TLSを有効にする必要があります。ピアのTLS許可を作成する方法の説明については、 [Registering and enrolling identities with a CA](https://hyperledger-fabric-ca.readthedocs.io/en/release-1.4/deployguide/use_CA.html) にあるトピックを参照してください。

Reviewer: ```suggestion
ピアに対してTLSコミュニケーションを有効にするために、このセクションを設定します。TLSが有効になった後は、ピアとやりとりする全てのノードも、TLSを有効にする必要があります。ピアのTLS証明書を作成する方法の説明については、 [Registering and enrolling identities with a CA](https://hyperledger-fabric-ca.readthedocs.io/en/release-1.4/deployguide/use_CA.html) にあるトピックを参照してください。
```

- **`enabled`:** (デフォルト値を上書きする必要があります。) 本番環境が安全であることを保証するには、設定ファイルの `tls` セクションにある `enabled: true` を設定して、すべてのノード間の通信でTLSが有効になっている必要があります。このフィールドはデフォルトでは無効にされていますが、テストのネットワークとしては受け入れられるかもしれませんが、本番では有効にする必要があります。この設定は、 **server-side TLS** を構成し、TLSが _サーバー_ のアイデンティティをクライアントへ保証し、それらに双方向の暗号化されたチャネルを提供することを意味します。

> - **`cert.file`:** (デフォルト値を上書きする必要があります。) すべてのピアは、組織内の他のノードと安全に取引する前に、TLS CAに登録してエンロールする必要があります。そのため、ピアをデプロイする前に、まずピアのユーザーを登録し、ピアのアイデンティティをTLS CAにエンロールして、ピアのTLS署名付き証明書を生成する必要があります。 [Registering and enrolling identities with a CA](https://hyperledger-fabric-ca.readthedocs.io/en/release-1.4/deployguide/use_CA.html) トピックの推奨フォルダ構造を使用している場合は、このファイルを  `config/organizations/peerOrganizations/org1.example.com/peers/peer0.org1.example.com/tls` にコピーする必要があります。

Reviewer: ```suggestion
- **`cert.file`:** (デフォルト値を上書きする必要があります。) すべてのピアは、組織内の他のノードと安全にやりとりする前に、TLS CAに登録してエンロールする必要があります。そのため、ピアをデプロイする前に、まずピアのユーザーを登録し、ピアのアイデンティティをTLS CAにエンロールして、ピアのTLS署名付き証明書を生成する必要があります。 [Registering and enrolling identities with a CA](https://hyperledger-fabric-ca.readthedocs.io/en/release-1.4/deployguide/use_CA.html) トピックの推奨フォルダ構造を使用している場合は、このファイルを  `config/organizations/peerOrganizations/org1.example.com/peers/peer0.org1.example.com/tls` にコピーする必要があります。
```

- **`key.file`:** (デフォルト値を上書きする必要があります。) `cert.file` と同様に、このピアに対して生成されたTLS秘密鍵の名前と場所を指定します。例：  `/msp/keystore/87bf5eff47d33b13d7aee81032b0e8e1e0ffc7a6571400493a7c_sk`。 [Registering and enrolling identities with a CA](https://hyperledger-fabric-ca.readthedocs.io/en/release-1.4/deployguide/use_CA.html) トピックの推奨フォルダ構造を使用している場合は、このファイルを `config/organizations/peerOrganizations/org1.example.com/peers/peer0.org1.example.com/tls` にコピーする必要があります。 [HSM](#bccsp) を使用してピアの秘密鍵を格納する場合、このフィールドは空白になります。

- **`rootcert.file`:**  (デフォルト値を上書きする必要があります。) この値には、ピア組織のCAルート証明書の名前と場所が含まれています。 [Registering and enrolling identities with a CA](https://hyperledger-fabric-ca.readthedocs.io/en/release-1.4/deployguide/use_CA.html) トピックの推奨フォルダ構造を使用している場合は、このファイルを `config/organizations/peerOrganizations/org1.example.com/peers/peer0.org1.example.com/tls` にコピーする必要があります。

次の2つのパラメータは、相互TLSが必須の場合にのみ指定する必要があります。

> - **`clientAuthRequired`:** デフォルトは `false` です。 `true` に設定すると、 **mutual TLS** を使用してセキュリティレベルを高くすることができ、クライアント側のTLS証明書の追加の検証ステップとして設定できます。サーバー側のTLSが最低限必要なセキュリティレベルであると考えられる場合、相互TLSは追加で任意のセキュリティレベルです。

Reviewer: ```suggestion
- **`clientAuthRequired`:** デフォルトは `false` です。 `true` に設定すると、 **相互TLS** を使用してセキュリティレベルを高くすることができ、クライアント側のTLS証明書の追加の検証ステップとして設定できます。サーバー側のTLSが最低限必要なセキュリティレベルであると考えられる場合、相互TLSは追加で任意のセキュリティレベルです。
```
前段と統一したほうがよいでしょう。

       password:
```

> このセクションは、台帳のデータベースタイプ( `goleveldb` または `CouchDB` )を選択するために使用します。エラーを回避するには、すべてのピアが同じデータベース **type** を使用する必要があります。JSONクエリが必須である場合、CouchDBが適切です。CouchDBは別のオペレーティングシステムプロセスで実行されますが、ピアノードとCouchDBインスタンスの間には依然として1:1の関係があります。つまり、各ピアは1つのデータベースを持ち、そのデータベースはそのピアにのみ関連付けられます。CouchDBの追加のJSONクエリケーパビリティに加えて、データベースの選択はスマートコントラクトには見えません。

Reviewer: ```suggestion
このセクションは、台帳のデータベースタイプ( `goleveldb` または `CouchDB` )を選択するために使用します。エラーを回避するには、すべてのピアが同じデータベース **type** を使用する必要があります。JSONクエリが必須である場合、CouchDBが適切です。CouchDBは別のオペレーティングシステムプロセスで実行されますが、ピアノードとCouchDBインスタンスの間には依然として1:1の関係があります。つまり、各ピアは1つのデータベースを持ち、そのデータベースはそのピアにのみ関連付けられます。CouchDBの追加のJSONクエリケーパビリティ以外は、データベースの選択はスマートコントラクトには見えません。
```

            files: []
```

運用サービスはピアのヘルス状態を監視するために使用され、通信を安全にするために相互TLSに依存しています。そのため、 `operations.tls.clientAuthRequired` を `true` に設定する必要があります。このパラメータが `true` に設定されている場合、ノードの状態を確認しようとするクライアントは、有効な認証の証明書を提供する必要があります。クライアントが証明書を提供していない場合や、サービスがクライアントの証明書を確認できない場合は、要求は拒否されます。つまり、クライアントはピアのTLS CAに登録し、そのリクエストにTLS署名証明書を提供する必要があります。詳細については [The Operations Service](../operations_service.html) を参照してください。

Prometheus [metrics](#metrics) を使用してピアを監視する場合は、この運用サービスを設定する必要があります。

> 2つのピアが同じノードで実行されている可能性が低い場合は、2番目のピアのアドレスを変更して別のポートを使用する必要があります。そうでない場合、2番目のピアを開始すると、開始に失敗し、アドレスがすでに使用中であることが報告されます。

Reviewer: ```suggestion
あまりないことですが2つのピアが同じノードで実行されている場合は、2番目のピアのアドレスを変更して別のポートを使用する必要があります。そうでない場合、2番目のピアを開始すると、開始に失敗し、アドレスがすでに使用中であることが報告されます。
```
